pages/common/mosquitto_sub.md

```diff
@@ -0,0 +1,19 @@
+# mosquitto_sub
+
+> A simple MQTT version 3.1.1 client that will subscribe to topics and print the messages that it receives.
+
+- Subscribe to all broker status messages:
+
+`mosquitto_sub -v -t {{\$SYS/#}}`
```

Comment:
Are there any long forms of these options available? In tldr-pages, we prefer to use the long options where possible, as they are more descriptive.

Reply:
Updated